kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/validator/NoWhitespaceCharactersValidator.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.web.shared.validator;
+
+import java.util.function.Predicate;
+
+public class NoWhitespaceCharactersValidator extends PredicateValidator {
+
+    private static final Predicate<String> PREDICATE = value -> {
+        if (value == null) {
```

Comment:
is null a valid non whitespace string ?

Reply:
Good question.. I accepted a null string so that it will trigger the nonEmptyValidator instead of this one providing the "value cannot be empty" messages instead of "value cannot contain whitespace characters"

Reply:
Does it makes sense change the validators order and perform the nonEmptyValidator before NonWhitespaceCharactersValidator ?

Reply:
It should be already like that https://github.com/eclipse/kura/blob/3cd7aeb21827a773a827dcce770ae06387ad93b4/kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/validator/GwtValidators.java#L86, in any case a null or empty string does not contain any whitespace character